### PR TITLE
pick coarsest freq for scatter plot

### PR DESCRIPTION
**Pull request by charlienegri**
this should be consistent with what the map engine does now (after #816 was merged)

### CODE REVIEW

File: pyaerocom/aeroval/coldatatojson_engine.py [hunk 1]
```diff
@@ -225,6 +225,9 @@ def process_coldata(self, coldata: ColocatedData):
 
             _write_site_data(ts_objs, out_dirs["ts"])
 
+            scatter_freq = min(TsType(fq) for fq in self.cfg.time_cfg.freqs)
+            scatter_freq = str(min(scatter_freq, main_freq))
```
Comment (avaldebe) [inline pyaerocom/aeroval/coldatatojson_engine.py:229]: please keep the `scatter_freq` type to `TsType` here and convert to string on the `_process_map_and_scat` call (if needed).

**Review by avaldebe — CHANGES_REQUESTED** [review]
small nitpick regarding types/typing, otherwise good

File: pyaerocom/aeroval/coldatatojson_engine.py (hunk 1 above)
Comment (charlienegri) [reply pyaerocom/aeroval/coldatatojson_engine.py]: 👍

**Review by avaldebe — APPROVED** [review]
🦄 🔥 🚒

**Pull request merged**